css/properties/display.json

```diff
@@ -228,6 +228,38 @@
             }
           }
         },
+        "flow": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "115"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "≤72"
+              },
```

Comment:
According to https://bugzil.la/1657594, Firefox may not actually support it:

> The Compatibility panel should indicate that `display: flow` is converted to `display:block` in Firefox,

Reply:
Discussed with Vinyl, and the question is whether it's always `display:block`, or if it is converted to the default value for the element (e.g. `display: inline` for `<span>`)

Reply:
So I generated a test case with ChatGPT: https://developer.mozilla.org/de/play?id=h%2BLlW3ImqADHCJYkMw2xE%2BjFSvlfWApNyUj1hSyEa2VSWUgXOu84ObYg3ftPQZ74pE8GnTGvbE18Lg0V

Tested in Chrome 131, Firefox Nightly 134.0a1 (2024-11-21), and Safari 18.1.1.

All browsers do **not** _seem_ to support `flow`, i.e. the computed value is `block`.

But maybe that's in accordance to the spec:

> Otherwise it generates a [block container](https://drafts.csswg.org/css-display/#block-container) box.

So I came up with another (ChatGPT-generated) test case that sets `style.display` and reads it again: https://developer.mozilla.org/en-US/play?id=UGBq0JaWyjIGJz8nqIEbFux3a3rDR5mS3uppv7pl%2Bn03MxTNx0%2B%2BXNGDjtIzkD0ICNQbWIjuTJQKhQJm

And this test also fails in all three browsers engines. 🤔